README.md

```diff
@@ -329,6 +329,27 @@ Normally, setting required DSN information would be enough.
   ```
   - See https://docs.sentry.io/platforms/node/pluggable-integrations/ for more information
 
+### enableTracing
```

Comment:
We should probably mention that `@sentry/tracing` should be installed manually when using this option.

I guess it should work regardless because `@sentry/node` seems to depend on it but probably better to be more explicit.